includes/class-taxjar-refund-record.php

```diff
@@ -57,8 +57,8 @@ public function should_sync() {
 			return false;
 		}
 
-		if ( empty( $data[ 'from_country' ] ) || empty( $data[ 'from_state' ] ) || empty( $data[ 'from_zip' ] ) || empty( $data[ 'from_city' ] ) ) {
-			$this->add_error( __( 'Refund failed validation - missing required ship from field on parent order.', 'wc-taxjar' ) );
+		if ( ! $this->has_valid_ship_from_address() ) {
+			$this->add_error( __( 'Order failed validation, missing required ship from field', 'wc-taxjar' ) );
```

Comment:
@dallendalton Should this say `Refund failed validation` instead of `Order`?

Reply:
Ahh yes you are correct. That was not an intentional change and I will get it updated.